lib/recurly/resource/pager.rb

```diff
@@ -64,24 +64,32 @@ def count
         @count ||= API.head(uri, @options)['X-Records'].to_i
       end
 
-      # @return [Array] Iterates through the current page of records.
+      # @return [Array,Enumerator] Iterates through the current page of records.
       # @yield [record]
-      def each
-        return enum_for :each unless block_given?
+      # Yields each record of the current page.
+      def each_current_page
```

Comment:
might consider naming this [`each_slice`](https://ruby-doc.org/core-2.3.1/Enumerable.html#method-i-each_slice)